[IMP] email_marketing: getting started

samueljlieber opened the pull request:
This PR tests the push process of the BE doc team.

## The following process created this PR:

Updated base branch
```
git checkout 16.0
git pull
git log --oneline
```
Created feature branch
```
git switch -c 16.0-us-docteam-push-sali
```
Stage and commit edits
```
git add .
git commit -m "[IMP] email_marketing: getting started"
```
Push to remote
```
git push --force-with-lease --set-upstream origin 16.0-be-docteam-push-sali
```

samueljlieber committed:
[IMP] email_marketing: getting started

samueljlieber commented:
Hi @saliodootest this PR is ready for your review

saliodootest commented on content/applications/marketing/email_marketing.rst:
```diff
@@ -5,6 +5,8 @@
 Email marketing
 ===============
 
+This is the first change in the pr for BE push process. Howpeully it works good.
```
You have a serious problem.
```suggestion
This is the first change in the pr for BE push process. Hopefully it works well.
```

saliodootest reviewed (changes requested):
Ive seen better

samueljlieber commented:
I can `git pull --rebase` with no issues, updating my local with remote

samueljlieber commented:
Squashed commits and added a sentence in https://github.com/samueljlieber/dev-documentation/commit/2c8970adbe65e4c08aac61e9217aab272def16a7 using `git push --force-with-lease --set-upstream origin 16.0-be-docteam-push-sali`

saliodootest reviewed (approved):
Looks good to me

saliodootest commented on content/applications/marketing/email_marketing.rst:
```diff
@@ -5,6 +5,9 @@
 ===============
 
+This is the first change in the pr for BE push process. Hopefully it works well.
+I added this sentence because its cool okay?
```
Nope bad sentence i remove
```suggestion
```

saliodootest reviewed (commented):
Im removing this sentence

samueljlieber commented:
I am going to make a commit locally ignoring your commit here, and try to push up 😈

samueljlieber commented:
I recieved an error!!
```
git push --force-with-lease --set-upstream origin 16.0-be-docteam-push-sali

To github.com:samueljlieber/dev-documentation.git
 ! [rejected]            16.0-be-docteam-push-sali -> 16.0-be-docteam-push-sali (stale info)
error: failed to push some refs to 'github.com:samueljlieber/dev-documentation.git'
```

samueljlieber commented:
This was resolved by `git pull --rebase` fixed merge conflicts (due to same line edits)
